compiler/src/dmd/trace.d

```diff
@@ -521,7 +521,9 @@ struct TraceFileTail
 
 private bool isLikelyOnStack(in void* v) @safe pure nothrow @nogc
 {
-    pragma(inline, true);
+    // dmd seems to be unable to inline this function
+    // which is plausable since it uses asm
+    // pragma(inline, true);
```

Comment:
What's wrong with using this directive anyway? Does it make any harm keeping it qualified?

Reply:
Yes.
dmd will refuse to compile code when it cannot inline force-inline functions.

Reply:
I changed to

```d
version(LDC) pragma(inlline, true);
```

.